fe/fe-core/src/test/java/com/starrocks/analysis/CreateRoutineLoadStmtTest.java

```diff
@@ -104,6 +104,7 @@ public void testParser() {
                 + "FROM KAFKA\n"
                 + "(\n"
                 + "\"kafka_broker_list\" = \"kafkahost1:9092,kafkahost2:9092\",\n"
+                + "\"confluent.schema_registry.url\" = \"https://user:password@confluent.west.us\",\n"
```

Comment:
user and password may be not permitted exposed. Is there any other authorization style?

Reply:
This could be hided by adding the key word to `PrintableMap`.
https://github.com/StarRocks/starrocks/blob/c82cb09c3dc4bb0f8404bebaa4f70d61e0a5510a/fe/fe-core/src/main/java/com/starrocks/common/util/PrintableMap.java#L37-L48

Reply:
<img width="942" alt="image" src="https://user-images.githubusercontent.com/45813655/212804157-9c3f0b9a-0038-41eb-9a44-fec812fcfe4e.png">